doc/plugin_server_nodeattestor_gcp_iit.md

```diff
@@ -61,5 +62,7 @@ corresponding selector will still have a trailing colon (i.e.
 `gcp_iit:label:<key>:`, `gcp_iit:metadata:<key>:`)
 
 ## Authenticating with the Google Compute Engine API
+The plugin uses the Application Default Credentials to authenticate with the Google Compute Engine API, as documented by [Setting Up Authentication For Server to Server](https://cloud.google.com/docs/authentication/production). When SPIRE Server is running inside GCP, it will use the default service account credentials available to the instance it is running under. When running outside GCP, or if non-default credentials are needed, the path to the service account file containing the credentials may be specified using the GOOGLE_APPLICATION_CREDENTIALS environment variable or the service_account_file configurable (see Configuration).
```

Comment:
should probably surround GOOGLE_APPLICATION_CREDENTIALS and `service_account_file` in backticks to be consistent